src/main/webapp/history.js

```diff
@@ -82,6 +82,8 @@ function createTutoringSessionBox(tutoringSession) {
     var minute = parseInt(tutoringSession.timeslot.start) % 60;
     if (minute == 0) {
         minute = "00";
+    } else if (minute < 10) {
+        minute = "0" + minute;
```

Comment:
I think you can get rid of the if statement and just keep the else if and it will work the same.

Reply:
True! Thanks